Tests/FluentAssertions.Equivalency.Specs/DataRelationSpecs.cs

```diff
@@ -254,5 +254,101 @@ public void When_ChildColumns_do_not_match_and_property_is_excluded_it_should_su
                 .ExcludingRelated((DataRelation dataRelation) => dataRelation.ParentKeyConstraint)
                 .ExcludingRelated((DataRelation dataRelation) => dataRelation.ChildKeyConstraint));
         }
+
+        [Fact]
+        public void When_object_of_type_unequal_to_DataRelation_is_asserted_with_a_DataRelation_it_fails()
```

Comment:
🔧 In addition what @jnyrup said earlier, also stay away from using the literal names of code elements like `DataRelation`, `DataSet` and such in test names. Keep them functional.